lib/hooks/useCommonData/useCommonData.js

```diff
@@ -20,22 +20,22 @@ import {
 } from '../../queries';
 import { isConsortiaEnv } from '../../utils';
 
-const useCommonData = () => {
+const useCommonData = (tenantId) => {
   const stripes = useStripes();
 
   const { consortiaTenants, isLoading: isConsortiaTenantsLoading } = useConsortiaTenantsQuery();
-  const { locationsFromAllTenants, isConsolidationLocationLoading } = useLocationsFromAllTenantsQuery({ consortiaTenants });
-  const { locations, isLoading: isLocationsLoading } = useLocationsQuery({ enabled: !isConsortiaEnv(stripes) });
-  const { materialTypes, isLoading: isMaterialTypesLoading } = useMaterialTypesQuery();
-  const { natureOfContentTerms, isLoading: isNatureOfContentTermsLoading } = useNatureOfContentTermsQuery();
-  const { holdingsTypes, isLoading: isHoldingsTypesLoading } = useHoldingsTypesQuery();
-  const { statisticalCodeTypes, isLoading: isStatisticalCodeTypesLoading } = useStatisticalCodeTypesQuery();
-  const { statisticalCodes, isLoading: isStatisticalCodesLoading } = useStatisticalCodesQuery();
-  const { modesOfIssuance, isLoading: isModesOfIssuanceLoading } = useModesOfIssuanceQuery();
-  const { instanceStatuses, isLoading: isInstanceStatusesLoading } = useInstanceStatusesQuery();
-  const { instanceFormats, isLoading: isInstanceFormatsLoading } = useInstanceFormatsQuery();
-  const { holdingsSources, isLoading: isHoldingsSourcesLoading } = useHoldingsSourcesQuery();
-  const { instanceTypes, isLoading: isInstanceTypesLoading } = useInstanceTypesQuery();
+  const { locationsFromAllTenants, isConsolidationLocationLoading } = useLocationsFromAllTenantsQuery({ consortiaTenants, tenantId });
+  const { locations, isLoading: isLocationsLoading } = useLocationsQuery({ enabled: !isConsortiaEnv(stripes), tenantId });
+  const { materialTypes, isLoading: isMaterialTypesLoading } = useMaterialTypesQuery(tenantId);
+  const { natureOfContentTerms, isLoading: isNatureOfContentTermsLoading } = useNatureOfContentTermsQuery(tenantId);
+  const { holdingsTypes, isLoading: isHoldingsTypesLoading } = useHoldingsTypesQuery(tenantId);
+  const { statisticalCodeTypes, isLoading: isStatisticalCodeTypesLoading } = useStatisticalCodeTypesQuery(tenantId);
+  const { statisticalCodes, isLoading: isStatisticalCodesLoading } = useStatisticalCodesQuery(tenantId);
+  const { modesOfIssuance, isLoading: isModesOfIssuanceLoading } = useModesOfIssuanceQuery(tenantId);
+  const { instanceStatuses, isLoading: isInstanceStatusesLoading } = useInstanceStatusesQuery(tenantId);
+  const { instanceFormats, isLoading: isInstanceFormatsLoading } = useInstanceFormatsQuery(tenantId);
+  const { holdingsSources, isLoading: isHoldingsSourcesLoading } = useHoldingsSourcesQuery(tenantId);
+  const { instanceTypes, isLoading: isInstanceTypesLoading } = useInstanceTypesQuery(tenantId);
```

Comment:
I'd suggest keeping the current pattern to provide an options object in this case

Reply:
updated it

Reply:
https://github.com/user-attachments/assets/7c028772-c4f1-4b88-8936-9eef36100188